tests/components/zwave_js/test_update.py

```diff
@@ -310,7 +311,8 @@ async def test_update_entity_ha_not_running(
     hass_ws_client: WebSocketGenerator,
 ) -> None:
     """Test update occurs only after HA is running."""
-    await hass.async_stop()
+    with patch.object(hass.import_executor, "shutdown"):
+        await hass.async_stop()
```

Comment:
These tests setup an integration after stopping Home Assistant. This is a bit unexpected